src/lighteval/tasks/lighteval_task.py

```diff
@@ -91,7 +91,7 @@ class LightevalTaskConfig:
     prompt_function: Callable  # [[dict, str], Doc]
     hf_repo: str
     hf_subset: str
-    metric: Tuple[Union[str, Metrics]]
+    metric: Tuple[Union[Metric, Metrics]]
```

Comment:
in what case can we have a metrics enum instead of a metric ?

Reply:
If we pass Metrics.name_of_the_metric, it's different from passing a Metric